### PR TITLE
5810 DOD

### PR DESCRIPTION
**Pull request by mrinsin**
added tests around navigation from different qc sections, fixed problem discovered by sonar cloud (caused due to automatic merge)

### CODE REVIEW

File: web-client/integration-tests/journey/petitionsClerkReviewsPetitionAndSavesForLater.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export const petitionsClerkReviewsPetitionAndSavesForLater = test => {
+  return it('Petitions Clerk reviews petiton and saves for later', async () => {
```
Comment (rachaelparris) [inline web-client/integration-tests/journey/petitionsClerkReviewsPetitionAndSavesForLater.js]: Nit: typo `petition`

File: web-client/integration-tests/petitionsClerkSavesDocumentQCForLater.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    jest.setTimeout(30000);
+
+    global.window.localStorage.getItem = key => {
+      if (key === 'scannerSourceIndex') {
+        return `"${scannerSourceIndex}"`;
+      }
+
+      if (key === 'scannerSourceName') {
+        return `"${scannerSourceName}"`;
+      }
```
Comment (rachaelparris) [inline web-client/integration-tests/petitionsClerkSavesDocumentQCForLater.test.js]: Is this setup necessary here? I don't see anything in this test that should be scanning.

File: web-client/integration-tests/petitionsClerkSavesDocumentQCForLater.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+      return null;
+    };
+  });
+
+  loginAs(test, 'petitioner@example.com');
+  petitionerChoosesProcedureType(test, { procedureType: 'Regular' });
+  petitionerChoosesCaseType(test);
+  petitionerCreatesNewCase(test, fakeFile, { caseType: CASE_TYPES_MAP.cdp });
```
Comment (rachaelparris) [inline web-client/integration-tests/petitionsClerkSavesDocumentQCForLater.test.js]: Nit: this is actually quite a bit slower than using `uploadPetition` in `helpers.js` - I would prefer to see `uploadPetition` used instead unless testing these scenarios is necessary.

**Review by rachaelparris — APPROVED** [review]

File: web-client/integration-tests/petitionsClerkSavesDocumentQCForLater.test.js (hunk 3 above)
Comment (mrinsin) [reply web-client/integration-tests/petitionsClerkSavesDocumentQCForLater.test.js]: Thank you!

**Pull request merged**